File: scripts/problems/valid-palindrome/boilerplate-full.js
const lines = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
let _idx = 0;
const nextLine = () => lines[_idx++];

// ── USER CODE START ──
{{USER_CODE}}
// ── USER CODE END ──

const s = nextLine();

const result = isPalindrome(s);
console.log(result ? "true" : "false");

File: scripts/problems/valid-palindrome/boilerplate-full.rs
use std::io::{self, BufRead};

fn next_line() -> String {
    let stdin = io::stdin();
    let mut lines = stdin.lock().lines();
    lines.next().unwrap().unwrap()
}

// ── USER CODE START ──
{{USER_CODE}}
// ── USER CODE END ──

fn main() {
    let s: String = next_line().trim().to_string();

    let result = isPalindrome(s);
    println!("{}", if result { "true" } else { "false" });
}